go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/abrander/garmin-connect
+
+go 1.16
```

Comment:
How about we try requiring 1.15, and not 1.16? I believe older versions would suffice too, but I have no easy way of testing it.

Reply:
I will change it to 1.15. It is autogenerated by go mod init.